security/class-lockout.php

```diff
@@ -55,12 +55,12 @@ private function get_lockout_message() {
 		if ( defined( 'VIP_ACCOUNT_STATUS' ) && constant( 'VIP_ACCOUNT_STATUS' ) !== self::ACCOUNT_STATUS_NORMAL ) {
 			switch ( $this->get_lockout_state() ) {
 				case self::ACCOUNT_STATUS_WARNING:
-					return 'Payment for this WordPress VIP account is overdue and access will be disabled.<br />
+					return 'Payment for this WordPress VIP account is overdue and access will be suspended soon.<br />
 Please contact accounts@wpvip.com to settle your bill.';
```

Comment:
Should we also make the email here a link?

Reply:
Oh i didn't see this until just now, I asked the same thing 😂 ... yes I think we should.

Reply:
And, done :)